tools/ci/modal_nightly.py

```
@@ -0,0 +1,292 @@
+        run(["git", "init", str(WORKDIR)])
+        run(["git", "remote", "add", "origin", REPO_URL], cwd=WORKDIR)
+        run(["git", "fetch", "--depth", "1", "origin", ref], cwd=WORKDIR)
+        run(["git", "checkout", "--detach", "FETCH_HEAD"], cwd=WORKDIR)
+        return
+
+    run(["git", "clone", "--depth", "1", "--branch", ref, REPO_URL, str(WORKDIR)])
+
+
+def run_test_target(target: str) -> None:
+    if target == "test_nightly":
+        has_target = subprocess.run(
```

Ruff (0.15.15)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)